recipes-connectivity/awsclient/awsclient/awsclientcheck.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+FILE="config/esec.config"
+
+if [ -f "${FILE}" ]; then
+    condition=$(jq .awsclient < $FILE | sed -r 's/["]+//g')
+    [ "$condition" = start ] && awsclient
+fi
+exit 0
```

Comment:
This is not ideal, as this hides the exit code of the awsclient.

Reply:
is changed